generate.go

```diff
@@ -0,0 +1,32 @@
+package cogs
+
+import (
+	"fmt"
+	"github.com/BurntSushi/toml"
+)
+
+type CogManifest struct {
+	Name string
+	Envs map[string]Env
+}
+
+type Env struct {
+	Name string
+	Cfgs []Cfg
+}
+
+type Cfg struct {
```

Comment:
all the fields within this struct are private, does this need to be public?

Reply:
It does not, I find when starting from scratch making most fields public makes it easier to iterate and making them private later, perhaps it's a bad habit though.

Reply:
I noticed its used on PR #2, so just to record it here, ignoring this file for this review.